### ISSUE TITLE
Pagination Windows settings

### ISSUE THREAD

**Issue by iamramit** [javve/list.js#599]
I need to set the pagination so that if i have 16 pages each 10 records per page, 
I set the options of pagination to , 
pagination: { 
innerWindow: 2,
left: 1, 
outerWindow: 2,
right: 1},

What i see with these settings is 
When on First page, i see, 
<img width="396" alt="screen shot 2018-05-31 at 7 51 08 pm" src="https://user-images.githubusercontent.com/1033280/40787824-29ebc5d8-650c-11e8-87ab-e0b547ab7419.png">

When on the middle I see
<img width="554" alt="screen shot 2018-05-31 at 7 51 22 pm" src="https://user-images.githubusercontent.com/1033280/40787857-3bdb531c-650c-11e8-87c5-c1218b7a5601.png">

When on the last i see:
<img width="431" alt="screen shot 2018-05-31 at 7 51 30 pm" src="https://user-images.githubusercontent.com/1033280/40787867-440c2444-650c-11e8-933d-d9cb23d9c2c7.png">

So what i actually i want to see is 👍 

Beginning: 
<img width="452" alt="screen shot 2018-05-31 at 7 50 01 pm" src="https://user-images.githubusercontent.com/1033280/40787890-5189f934-650c-11e8-98f9-bf01253da8ea.png">

Middle:
<img width="458" alt="screen shot 2018-05-31 at 7 50 12 pm" src="https://user-images.githubusercontent.com/1033280/40787911-5bd7dfaa-650c-11e8-9074-21a8f1baf430.png">

Last: 
<img width="498" alt="screen shot 2018-05-31 at 7 50 17 pm" src="https://user-images.githubusercontent.com/1033280/40787927-67ac3bfa-650c-11e8-9097-229129836ea7.png">

so basically i need to see First 5 only when i am on beginning of them , and on the last vice-versa,
I know this can be achieved with left: 
but it stays there, even when i reach middle.

Please support me with your answer.
Thank you in advance

**Comment by mcglonelevi** [comment]
I don't think this is possible using default pagination from what I have read.

**Comment by javve** [comment]
This is an interesting feature request.
I've added it to the [wishlist](https://github.com/javve/list.js/blob/master/WISHLIST.md) and I may take it in account next time I work with the pagination. I'll close this issue until then.
